src/pages/home/report/ParticipantLocalTime.js

```diff
@@ -42,14 +42,15 @@ class ParticipantLocalTime extends PureComponent {
 
     getParticipantLocalTime() {
         const reportRecipientTimezone = lodashGet(this.props.participant, 'timezone', CONST.DEFAULT_TIME_ZONE);
-        moment.locale(this.props.preferredLocale);
-        const reportRecipientDay = moment().tz(reportRecipientTimezone.selected).format('dddd');
-        const currentUserDay = moment().tz(this.props.currentUserTimezone.selected).format('dddd');
+        const reportTimezone = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, null, reportRecipientTimezone.selected);
+        const currentTimezone = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, null, this.props.currentUserTimezone.selected);
```

Comment:
Since the default value of the third param is `currentSelectedTimezone = timezone.selected` and it has the same value as being passed here, let's remove `this.props.currentUserTimezone` from this component completely and clean things up a bit.

Reply:
done!

Reply:
Ah thanks! I was thinking it would be a little more than just that... I believe that prop is no longer used on that component so the propTypes can be cleaned up and then the prop needs to be removed from wherever this is rendered. I believe it's in the report action compose component. Once the prop is removed from there, I think you can remove one of the Onyx bindings from that component also.

Reply:
Agree, I forgot to clean up the props because that prop is not defined in propTypes.
Also, I think that Onyx binding will still remain in `ReportActionCompose.js`.. since it's being used on line 481 as well
https://github.com/Expensify/App/blob/25e2ec4212a884278b4f27508be91e924f0ee406/src/pages/home/report/ReportActionCompose.js#L480-L481

Reply:
Ah yes, you are right about the Onyx binding. Good catch!